assets/css/style.css

```diff
@@ -1,137 +1,157 @@
-@import url('https://fonts.googleapis.com/css2?family=PT+Sans&family=Ubuntu:wght@300&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Dancing+Script:wght@500&family=PT+Sans&family=Ubuntu:wght@300&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans&family=Ubuntu:wght@300&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Dancing+Script:wght@500&family=PT+Sans&family=Ubuntu:wght@300&display=swap");
 
 /*BASE STYLING*/
 /*variable for color and text to be used accross the web*/
 :root {
```

Comment:
great use of pseudo-classes! These will be super key to creating scalable apps later down the line :-)

Reply:
Thank you :-)